gandalf.js

```diff
@@ -1,50 +1,48 @@
-const randomIndex = Math.floor(Math.random() * array.length);
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.